File: 7-kyu/can-this-object-fly--balloons-in--up--and-in-real-life/javascript/solution.js
class Journey {
  constructor(object, crew, balloons) {
    this.weight = object.weight;
    this.crew = crew;
    this.balloons = balloons;
  }
  isPossible() {
    return this.weight + this.crew * 80 <= this.balloons * 0.0048;
  }
}

File: 7-kyu/can-this-object-fly--balloons-in--up--and-in-real-life/python/solution.py
class Journey:
    def __init__(self, weight, crew, balloons):
        self.weight = weight['weight']
        self.crew = crew
        self.balloons = balloons
        self.CREW_WEIGHT = 80
        self.BALLOON_LIFT = 0.0048
    
    def isPossible(self):
        return self.weight + self.crew * self.CREW_WEIGHT <= self.balloons * self.BALLOON_LIFT
